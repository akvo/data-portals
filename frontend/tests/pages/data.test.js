import React from 'react'
import { render } from '@testing-library/react'
import Data from '../../pages/data'

describe('Data page', () => {
  it('matches snapshot', () => {
    const { asFragment } = render(<Data />, {})
    expect(asFragment()).toMatchSnapshot()
  })
})
