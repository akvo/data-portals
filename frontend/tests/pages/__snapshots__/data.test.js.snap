// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Data page matches snapshot 1`] = `
<DocumentFragment>
  <h1>
    Data
  </h1>
</DocumentFragment>
`;
